src/select/src/Select.tsx

```diff
@@ -560,25 +618,18 @@ export default defineComponent({
       }
       const { value } = e.target as unknown as HTMLInputElement
       patternRef.value = value
-      const { tag, remote } = props
+      const { tag, remote, multiple, separators } = props
       doSearch(value)
       if (tag && !remote) {
         if (!value) {
           beingCreatedOptionsRef.value = []
           return
         }
-        const optionBeingCreated = props.onCreate(value)
-        if (
-          compitableOptionsRef.value.some(
-            (option) => option.value === optionBeingCreated.value
-          ) ||
-          createdOptionsRef.value.some(
-            (option) => option.value === optionBeingCreated.value
-          )
-        ) {
-          beingCreatedOptionsRef.value = []
+        if (multiple && Array.isArray(separators) && ~separators.length) {
```

Comment:
```suggestion
        if (multiple && Array.isArray(separators) && ~separators.length) {
```
`separators.length` 永远是真值，因为 length 不可能等于 -1